src/crimson/common/interruptible_future.h

```diff
@@ -830,6 +862,14 @@ struct interruptor
 	      seastar::future<>>(seastar::now());
   }
 
+  template <typename ValueT = void, typename... A>
+  [[gnu::always_inline]]
+  static interruptible_future_detail<InterruptCond, seastar::future<ValueT>>
+  make_ready_future(A&&... value) {
```

Comment:
i don't think we need to support the variadic future anymore. let's just use `future<tuple>`.

Reply:
I used std::tuple inside the make_ready_future, leaving this method still accepting A&&..., I think this would save the caller to call std::make_tuple every time.

Reply:
@tchaikov done:-)